lib/events-watcher.js

```diff
@@ -22,7 +22,8 @@ function watchForEvents(fileName, events, log) {
 	  	log.error('Error tailing live event', data);
 	});
 	tail.watch();
+  events.finish = () => { tail.unwatch(); }
```

Comment:
Indentation here seems off from the rest of the file.

Reply:
Ah, tab vs spaces ... it was proper indented on idea IDE